packages/core/src/codewhisperer/models/constants.ts

```diff
@@ -280,66 +280,97 @@ export const newCustomizationMessage = 'You have access to new CodeWhisperer cus
+// Amazon Q Code Transformation
+
+export const amazonQFeedbackKey = 'Amazon Q'
+
+export const amazonQFeedbackText = 'Submit feedback'
 
 export const selectProjectPrompt = 'Select the project you want to transform'
 
+export const compilingProjectMessage = 'Amazon Q is compiling your project. This can take up to 10 minutes.'
+
+export const submittedProjectMessage =
+    'Your project has been submitted for transformation. The code transformation process may take 10-30 mins depending on the size of your project.'
+
 export const unsupportedJavaVersionSelectedMessage =
-    'Thank you for trying our product. We currently only support Java 8 and 11. We appreciate you taking the time to use the product, and hope to expand support to more Java versions in the future. Please see: https://docs.aws.amazon.com/amazonq/latest/aws-builder-use-ug/code-transformation.html#prerequisites'
+    'None of your open projects are supported by Amazon Q Code Transformation. For more information, see the [Amazon Q documentation](LINK_HERE).'
 
 export const transformByQWindowTitle = 'Amazon Q Code Transformation'
 
+export const failedToStartJobMessage =
+    "Amazon Q couldn't begin the transformation. Try starting the transformation again."
+
+export const failedToCompleteJobMessage =
+    "Amazon Q couldn't complete the transformation. Try starting the transformation again."
+
 export const stopTransformByQMessage = 'Stop transformation?'
 
 export const stoppingTransformByQMessage = 'Stopping transformation...'
 
-export const transformByQFailedMessage = 'Transformation failed'
-
-export const transformByQCancelledMessage = 'Transformation cancelled'
+export const transformByQCancelledMessage = 'You cancelled the transformation.'
 
-export const transformByQCompletedMessage = 'Transformation completed'
+export const transformByQCompletedMessage =
+    'Amazon Q successfully transformed your code. You can download a summary of the transformation and a diff with proposed changes in the Transformation Hub.'
 
-export const transformByQPartiallyCompletedMessage = 'Transformation partially completed'
+export const transformByQPartiallyCompletedMessage =
+    'Amazon Q partially transformed your code. You can download a summary of the transformation and a diff with proposed changes in the Transformation Hub.'
 
 export const noPomXmlFoundMessage =
-    'None of your open Java projects are supported by Amazon Q Code Transformation. We were unable to find a pom.xml in any of your Java projects. We only support Java projects built on Maven at the moment. Please see: https://docs.aws.amazon.com/amazonq/latest/aws-builder-use-ug/code-transformation.html#prerequisites'
+    'None of your open Java projects are supported by Amazon Q Code Transformation. Currently, Amazon Q can only upgrade Java projects built on Maven. A pom.xml must be present in the root of your project to upgrade it. For more information, see the [Amazon Q documentation](LINK_HERE).'
```

Comment:
Not a blocker: I don't see the find/replace as a problem. Just wondering if there are examples of "delimiters" for this in the repo. Eg) `<replaceMe>` or `__link_here__`. Just something to help identify the find/replace better.

I think if this was dynamic it may be important. Thoughts?